olm-catalog/serverless-operator/project.yaml

```diff
@@ -6,19 +6,19 @@ project:
   # all components in `dependencies.previous` to the same versions as `dependencies` in the same PR.
   # Otherwise, the upgrade tests will not pass, as we have a different SO version with the same bundle contents.
   # Also make sure to update values under `olm.previous` by copying from `olm.replaces` and `olm.skipRange`.
-  version: 1.29.0
+  version: 1.30.0
 
 olm:
-  replaces: 1.28.0
-  skipRange: '>=1.28.0 <1.29.0'
+  replaces: 1.29.0
+  skipRange: '>=1.29.0 <1.30.0'
   channels:
     default: 'stable'
     list:
       - 'stable'
-      - 'stable-1.29'
+      - 'stable-1.30'
```

Comment:
I'm not entirely sure on the channel changes (like should we replace stable-1.29 or add stable-1.30?), so I'd leave that to others more familiar with the new release process or maybe you can describe why this would work?

Reply:
That'll add a label to the operator image. This's touching the same area from [my comment](https://github.com/openshift-knative/serverless-operator/pull/2073#discussion_r1193450198).

Per our slack chat, we can follow-up on the channel names in another PR. And get this merged to unblock operand bumps.

Reply:
I.e.: adding another entry produces the expected labels:
```
diff --git a/olm-catalog/serverless-operator/metadata/annotations.yaml b/olm-catalog/serverless-operator/metadata/annotations.yaml
index 552c66acb..46aa81b07 100644
--- a/olm-catalog/serverless-operator/metadata/annotations.yaml
+++ b/olm-catalog/serverless-operator/metadata/annotations.yaml
@@ -1,6 +1,6 @@
 annotations:
   operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.bundle.channels.v1: stable,stable-1.30
+  operators.operatorframework.io.bundle.channels.v1: stable,stable-1.30,stable-1.29
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
diff --git a/olm-catalog/serverless-operator/project.yaml b/olm-catalog/serverless-operator/project.yaml
index 8c7da92d7..d1209c5a5 100644
--- a/olm-catalog/serverless-operator/project.yaml
+++ b/olm-catalog/serverless-operator/project.yaml
@@ -16,6 +16,7 @@ olm:
     list:
       - 'stable'
       - 'stable-1.30'
+      - 'stable-1.29'
   previous:
     replaces: 1.28.0
     skipRange: '>=1.28.0 <1.29.0'
```